src/structures/Message.js

```diff
@@ -426,6 +426,26 @@ class Message extends Base {
     });
   }
 
+  /**
+   * Crossposts the message.
+   * @returns {Promise<Message>}
+   * @example
+   * // Crosspost a message
+   * if (message.channel.type === 'news') {
+   *    message.crosspost()
+   *      .then(msg => console.log('Crossposted message`))
+   *      .catch(console.error);
```

Comment:
This should have two spaces of indentation, however, there are 3 from the start of the line 434 and 435:
```suggestion
   *   message.crosspost()
   *     .then(msg => console.log('Crossposted message`))
   *     .catch(console.error);
```